stages/org.osbuild.systemd-journald

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python3
+"""
+Configures systemd-journald
+
+The 'config' option allows to create a systemd-journald configuration drop-in
+file in `/usr/lib/systemd/journald.conf.d` with the name `filename`.
+
+Drop-in configuration files can specify the following subset of options:
+- 'Journal' section
+  -  'Storage' option
+  -  'Compress' option
+  -  'SplitMode' option
+  -  'MaxFileSec' option
+  -  'MaxRetentionSec' option
+  -  'SyncIntervalSec' option
+  -  'Audit' option
+
+At least one option must be specified in the 'Journal' section.
+"""
+
+import configparser
+import os
+import sys
+
+import osbuild.api
+
+SCHEMA = r"""
+"additionalProperties": false,
+"properties": {
+    "filename": {
+        "type": "string",
+        "description": "Name of the systemd-journald drop-in file.",
+        "pattern": "^([[a-zA-Z0-9_]|\\.|-]){1,250}\\.conf$"
+    },
+    "config": {
+        "additionalProperties": false,
+        "type": "object",
+        "description": "systemd-journald configuration.",
+        "minProperties": 1,
+        "properties": {
+            "Journal": {
+                "additionalProperties": false,
+                "type": "object",
+                "description": "'Journal' configuration section.",
+                "minProperties": 1,
+                "properties": {
+                    "Storage": {
+                        "type": "string",
+                        "description": "Controls where to store journal data.",
+                        "pattern": "^(volatile|persistent|auto|none)$"
+                    },
+                    "Compress": {
+                        "type": "string",
+                        "description": "Sets whether the data objects stored in the journal should be compressed or not. Can also take threshold values.",
+                        "pattern": "^(yes|no|[0-9]+(K|M|G))$"
+                    },
+                    "SplitMode": {
+                        "type": "string",
+                        "description": "Splits journal files per user or to a single file.",
+                        "pattern": "^(uid|none)$"
+                    },
+                    "MaxFileSec": {
+                        "type": "string",
+                        "description": "Max time to store entries in a single file. By default seconds, may be sufixed with units to override this.",
+                        "pattern": "^[0-9]+(year|month|week|day|h|m){0,1}$"
+                    },
+                    "MaxRetentionSec": {
+                        "type": "string",
+                        "description": "Maximum time to store journal entries.",
+                        "pattern": "^[0-9]+(year|month|week|day|h|m){0,1}$"
+                    },
+                    "SyncIntervalSec": {
+                        "type": "integer",
+                        "minimum": 0,
+                        "description": "Timeout before synchronizing journal files to disk."
```

Comment:
This takes the same arguments as the other `*Sec` configuration options. I would just keep this a "string" like the others, without a minimum and without a pattern.